blocks/address/src/app.tsx

```diff
@@ -0,0 +1,677 @@
+    <ThemeProvider theme={theme}>
+      <Box
+        ref={blockRootRef}
+        sx={{ display: "inline-block" }}
+        onMouseEnter={() => setHovered(true)}
+        onMouseLeave={() => setHovered(false)}
+      >
+        <Fade
+          in={hovered || autocompleteFocused || animatingIn || animatingOut}
+        >
+          <Box sx={{ display: "flex", columnGap: 3, flexWrap: "wrap" }}>
+            <Link
+              //  @todo: link this to the block's hub page
+              href=""
+              target="_blank"
+              variant="regularTextLabels"
+              sx={({ palette }) => ({
+                display: "inline-flex",
+                alignItems: "center",
+                textDecoration: "none",
+                fontSize: 15,
+                lineHeight: 1,
+                letterSpacing: -0.02,
+                marginBottom: 1.5,
+                whiteSpace: "nowrap",
+                color: palette.gray[50],
+                fill: palette.gray[40],
+                ":hover": {
+                  color: palette.gray[60],
+                  fill: palette.gray[50],
+                },
+              })}
+            >
+              Get help{" "}
+              <FontAwesomeIcon
+                icon={faQuestionCircle}
+                sx={{ fontSize: 16, ml: 1, fill: "inherit" }}
+              />
+            </Link>
+
+            <Typography
+              variant="regularTextLabels"
+              sx={{
+                display: "inline-flex",
+                alignItems: "center",
+                textDecoration: "none",
+                fontSize: 15,
+                lineHeight: 1,
+                letterSpacing: -0.02,
+                marginBottom: 1.5,
+                whiteSpace: "nowrap",
+                color: ({ palette }) => palette.gray[50],
+              }}
+            >
+              Using
+              {!selectedAddress ? (
+                <>
+                  <Box
+                    component="span"
+                    sx={{
+                      display: "inline-flex",
+                      alignItems: "center",
+                      color: ({ palette }) => palette.gray[60],
+                      mx: 1,
+                    }}
+                  >
+                    <MapboxIcon sx={{ fontSize: 16, mr: 0.375 }} />
+                    Mapbox Address Autofill{" "}
+                  </Box>
+                  and
+                </>
+              ) : null}
+              <Box
+                component="span"
+                sx={{
+                  display: "inline-flex",
+                  alignItems: "center",
+                  color: ({ palette }) => palette.gray[60],
+                  mx: 1,
+                }}
+              >
+                <MapboxIcon sx={{ fontSize: 16, mr: 0.375 }} />
+                Mapbox Static Images
+              </Box>
+              to render a fixed map
+            </Typography>
+          </Box>
+        </Fade>
+
+        <Collapse
```

Comment:
This probably shouldn't appear in `readonly` mode

Reply:
We should probably have an empty state for when the block is in readonly mode and without an address selected?

Reply:
I think potentially we should have nothing displayed. 

At least for sites like WordPress, 'readonly' is the state you want to show when displaying a page for information only, e.g. in this case simply the map and address. If there isn't a map and address to show, arguably you are better off showing nothing than some error-looking message about a map not being selected that the viewer can do nothing about.

I suggest we think about this further in follow-up since we need to do a v2 of the block next week anyway.